src/core/remote.js

```diff
@@ -2369,6 +2377,11 @@ Remote.prototype.requestPathFindClose = function(callback) {
 
   request.message.subcommand = 'close';
   request.callback(callback);
+  this._cur_path_find = null;   // TODO: wait for confirmation (RIPD-1013)
+  if (this._queued_path_finds.length > 0) {
```

Comment:
No, why do you ask? We're using `path_find` because rippled's implementation of `ripple_path_find` has an issue that makes it fail if a few simultaneous requests are made.